### ISSUE TITLE
Differences between template-loaded formulation and identical snapshot-imported molecule BB

### ISSUE THREAD

**Issue by AndreDlm** [Open-Systems-Pharmacology/PK-Sim#801]
Related to [#800](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/800):
Similar issues arise when repeating the procedure described in [#800](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/800) for different particle dissolution formulation types (monodisperse, polydisperse normal distribution, polydisperse lognormal distribution), e.g.:

![image](https://user-images.githubusercontent.com/26541603/40548256-7e17ca4e-6034-11e8-89d6-7c314dd27b0a.png)

Project file attached here: [#800](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/800):

**Comment by Yuri05** [comment]
@msevestre I think that's not a problem because visibility of particle parameters is properly handled once formulation is opened for editing?

**Comment by msevestre** [comment]
@Yuri05 That's right. Visibility is performed there. That being said, know that the code was moved outside of presenter, we could update the parameters when read from snapshot as well
